server/src/test/Parsing.test.ts

```diff
@@ -111,22 +134,36 @@ args {
             );
             done();
         });
-    });
 
-    describe('function with a Function object as argument', () => {
-        it('should parse correctly a function with a Function object as argument', (done) => {
+        it('should parse a function declaration with simple type arguments', (done) => {
             checkInputValidityForRule(
                 (parser) => parser.function(),
                 `
-function Test::applyMyFunction(Function testFunction)
---> action {
-    testFunction.apply();
-}
-;
-`,
+function LibDocument:printRangeValues(LibKPI:KeyPerformanceIndicator KPI,
```

Comment:
Hum… If `LibKPI` and co. are not public yet, consider turning some identifiers into gibberish for tests instead of revealing stuff. Maybe.

Reply:
Yeah, you're right. I did it for the last tests but didn't update the old ones.